File: PSET6/mario.py
from cs50 import get_int
answer = 0
while answer < 1 or answer > 8:
    answer = get_int("Enter Height:")
# if the answer keyed in is more than 2, do the following
if answer > 2:
    for i in range(0, answer):
        for j in range(7, i, -1):
            print(" ", end="")
        for k in range(-1, i):
            print("#", end="")
        print("\n", end="")     
# Else if the answer is 1, print one hash        
elif answer == 1:
    print("#")
# If the answer is exactly 2, print as per format required    
elif answer == 2:
    for l in range(1, 3):
        for s in range(2, l, -1):
            print(" ", end="")
        for m in range(0, l):
            print("#", end="")
        print("\n", end="")

File: PSET1/Mario/Mario.c
#include<cs50.h>
#include<stdio.h>

int main(void)
{
  
    int answer;
    
    do
    {
        answer = get_int("Height:");
    }
    
    //While condition to test if answer is between the range of 1 and 8
    while (answer < 1 || answer > 8);
    
    
    //If answer is 8, do the following
    if (answer == 8)
    {
    
        for (int i = 0; i < answer; i++)
        {
            for (int s = 7; s > i; s--)
            {
                printf(" ");
             
            }
       
            for (int k = -1; k < i; k++)
            {
                printf("#"); 
            }
       
            printf("\n");
 
        }
    
    }
    
    
    //If answer is 2, do the following
    else if (answer == 2)
    {
        for (int l = 1; l <= answer; l++)
        {
            
            for (int s = 2; s > l; s--)
            {
                printf(" ");
             
            } 
            for (int m = 0; m < l; m++)
            {
                printf("#");  
            }
            printf("\n");
        }
    }
    
    
    //If answer is 1, do the following
    else if (answer == 1)
    {
        printf("#\n"); 
    }
    
    
    //Otherwise, do the following
    else 
    {
        for (int l = 1; l <= answer; l++)
        {
       
            for (int m = 0; m < l; m++)
            {
                printf("#");  
            }
            
            printf("\n");
        }
    }

}



